### PR TITLE
OLM fixes

### PR DESCRIPTION
**Pull request by ecordell**
Fixing the ansible install, the cluster role and updating the names in the catalog.

### CODE REVIEW

File: roles/olm/files/olm-operator.clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:controller:operator-lifecycle-manager
+rules:
+- apiGroups: ["*"]
```
Comment (alecmerdler) [inline roles/olm/files/olm-operator.clusterrole.yaml]: :grin:

File: roles/olm/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+# do any asserts here
+
+- include_tasks: install.yaml
+  when: operator_lifecycle_manager_install | bool
+
+- include_tasks: remove.yaml
+  when: operator_lifecycle_manager_remove | bool
```
Comment (jpeeler) [inline roles/olm/tasks/main.yaml]: The pattern I've seen is setting one variable for this (instead using "when: not operator_lifecycle_manager_install").

Comment (ecordell) [reply roles/olm/tasks/main.yaml]: I'll follow up with that in the future - I want to make sure everything is installing correctly first, if that's okay

**Pull request merged**